src/containers/ProjectDashboardContainer.jsx

```diff
@@ -20,18 +22,37 @@ const propTypes = {
 };
 
 class ProjectDashboardContainer extends Component {
-
+  constructor() {
+    super();
+    this.handleSelect = this.handleSelect.bind(this);
+    this.state = {
+      translationLanguage: '',
+    };
+  }
   componentDidMount() {
     const { actions } = this.props;
     actions.fetchProject(this.props.params.project_id);
   }
 
+  handleSelect(event) {
+    this.setState({
+      translationLanguage: event.target.textContent,
+    });
+  }
+
   render() {
     const project = this.props.project.data;
     const { fieldguides, pages, workflows, tutorials } = this.props.project;
+    const renderSelectItems = languages.map(item => item.name);
```

Comment:
Mm, I think that line returns an array of strings 
![screen shot 2017-07-03 at 10 26 45](https://user-images.githubusercontent.com/8989606/27786727-3fddfcf4-5fda-11e7-8458-e1a0b35edbc4.png)